tools/mmp/Makefile

```diff
@@ -132,6 +132,7 @@ MMP_TARGETS = \
 	$(MAC_DESTDIR)$(MAC_FRAMEWORK_CURRENT_DIR)/lib/mmp/mmp.exe \
 	$(MAC_DESTDIR)$(MAC_FRAMEWORK_CURRENT_DIR)/lib/mmp/Mono.Cecil.dll \
 	$(MAC_DESTDIR)$(MAC_FRAMEWORK_CURRENT_DIR)/lib/mmp/Mono.Cecil.Mdb.dll \
+	$(MAC_DESTDIR)$(MAC_FRAMEWORK_CURRENT_DIR)/lib/mmp/Xamarin.Mac.registrar.mac.a \
```

Comment:
You'll need one static library per version of Xamarin.Mac.dll (which means one for mobile, one for 4.5 and one for full). This is because the generated code depends on metadata tokens, which is something that changes between the dlls.